src/wallet/rpc/backup.cpp

```diff
@@ -2036,6 +2038,14 @@ RPCHelpMan listdescriptors()
         if (active && type != std::nullopt) {
             spk.pushKV("internal", wallet->GetScriptPubKeyMan(true) == desc_spk_man);
         }
+        if (type != std::nullopt) {
+            std::string match = strprintf("/9'/%s'/0'", Params().ExtCoinType());
+            bool is_cj = descriptor.size() > 5 && descriptor.find(match) != std::string::npos;
+            if (is_cj) {
+                spk.pushKV("internal", false);
+                spk.pushKV("coinjoin", is_cj);
```

Comment:
Is there a possible risk of double-pushing `internal` due to this code?

https://github.com/dashpay/dash/blob/a12aca52b8d28179a7310731982097b43a64cba7/src/wallet/rpc/backup.cpp#L2038-L2040

If not, consider adding a `CHECK_NONFATAL(!active)`

Reply:
No, there's not possible to create duplicated keys; JSON requires uniqueness for keys.
Univalue as a JSON implementation does it too.
```
void UniValue::pushKV(const std::string& key, const UniValue& val_)
{   
    if (typ != VOBJ) throw std::runtime_error{"JSON value is not an object as expected"};

    size_t idx;
    if (findKey(key, idx))
        values[idx] = val_;
    else
        __pushKV(key, val_);
}  
```